### PR TITLE
test(answers): add unit tests for routes

### LINKED ISSUE #647: Add unit tests for Answers module
**Is your feature request related to a problem? Please describe.**

Unit tests are missing from the Answers module, meaning that it might be harder for us to track bugs when they occur.

**Describe the solution you'd like**

Unit tests for Answers module. 

**Additional context**

Blocked by #646.

### PR DESCRIPTION
**Pull request by LoneRifle**
## Problem and Solution

Add unit test coverage for `/answers` routes

Closes #647

### CODE REVIEW

**Review by wanlingt — APPROVED** [review]
lgtm

**Review by LinHuiqing — COMMENTED** [review]
for some reason, spec file isn't picked up by jest's coverage report

File: server/src/util/jest-db.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export enum ModelName {
  * Connect to a in-memory database
  */
 export const createTestDatabase = async (): Promise<Sequelize> => {
-  const sequelize = new Sequelize('sqlite::memory:', { logging: false })
+  const sequelize = new Sequelize('sqlite::memory:', { logging: true })
```
Comment (LinHuiqing) [inline server/src/util/jest-db.ts:30]: You forgot to switch logging back off...

Comment (LoneRifle) [reply server/src/util/jest-db.ts:30]: On it

**Pull request merged**